components/prism-system_verilog.js

```diff
@@ -0,0 +1,23 @@
+Prism.languages.system_verilog = {
+  'comment': /\/\/.*|\/\*[\w\W]*?\*\//,
+  'string': /"(\\\n|\\?.)*?"/,
+  // support for any kernel function (ex: $display())
+  'property': /(?!\b)\$\w+\b/,
```

Comment:
Why do you use a look-ahead here? Replacing `(?!\b)` with `\B` should do exactly the same result, I think...